tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -39,14 +38,55 @@ class FunctionOfTime;
 }  // namespace domain::FunctionsOfTime
 
 namespace {
-using Translation = domain::CoordinateMaps::TimeDependent::Translation;
-using Translation3D = domain::CoordinateMaps::TimeDependent::ProductOf3Maps<
-    Translation, Translation, Translation>;
 using BoundaryCondVector = std::vector<DirectionMap<
     3, std::unique_ptr<domain::BoundaryConditions::BoundaryCondition>>>;
 using Object = domain::creators::BinaryCompactObject::Object;
 using Excision = domain::creators::BinaryCompactObject::Excision;
 
+template <size_t Dim>
+struct TimeIndependentMetavariablesWithBoundaryCondition {
+  struct domain_metavariables
```

Comment:
I suggest to name this just `domain`, since the `_metavariables` suffix is redundant.